app/src/routes/api/v2/area.router.js

```diff
@@ -364,22 +400,31 @@ class AreaRouterV2 {
         ctx.statusCode = 204;
     }
 
-    static async getAlertsOfArea(ctx) {
-        logger.info(`Obtaining alerts of area with id ${ctx.params.id}`);
-        ctx.assert(ctx.query.precissionPoints, 400, 'precissionPoints is required');
-        ctx.assert(ctx.query.precissionBBOX, 400, 'precissionBBOX is required');
-        const result = await AreaModel.findOne({ _id: ctx.params.id });
-        if (!result) {
-            ctx.throw(404, 'Area not found');
+    static async updateByGeostore(ctx) {
+        if (ctx.state.loggedUser.role !== 'ADMIN') {
+            ctx.throw(401, 'Not authorized');
             return;
         }
-        let generateImages = true;
-        if (ctx.query.nogenerate) {
-            generateImages = false;
-        }
+        const geostores = ctx.request.body.geostores || [];
+        const updateParams = ctx.request.body.update_params || {};
+        logger.info('Updating geostores: ', geostores);
+        logger.info('Updating with params: ', updateParams);
+
+        // validate update json
+        updateParams.updatedDate = Date.now;
```

Comment:
Date.now is a function. you probably want the result of calling that function, no?

Reply:
Yes!